src/js/components/user-management/signup.js

```diff
@@ -27,6 +27,7 @@ export const Signup = ({ createOrganizationTrial, currentUserId, loginUser, setF
   const [organization, setOrganization] = useState('');
   const [tos, setTos] = useState(false);
   const [redirectOnLogin, setRedirectOnLogin] = useState(false);
+  const campaign = match?.params?.campaign;
```

Comment:
`match` and `params` should always be defined, due to the router wrapping the `Signup` component - but these conditions won't hurt anyone 🙂

Reply:
@mzedel they are not in unit tests, though... can I fix it somehow?